schema/deploy/warehouse/location.sql

```diff
@@ -0,0 +1,56 @@
+-- Deploy seattleflu/schema:warehouse/location to pg
+-- requires: warehouse/schema
+-- requires: postgis
+-- requires: hstore
+-- requires: citext
+
+begin;
+
+set local search_path to warehouse, public;
+
+create table location (
+    location_id integer primary key generated by default as identity,
+    identifier citext not null,
+    scale citext not null,
```

Comment:
This is a good question, and I deliberated on it for a while. Ultimately I decided no, but that may be the wrong decision. My thought for not restricting scale was that it allowed for more flexible handling of whatever scale was necessary for an analysis, without requiring creating of a row in a new controlled vocabulary table or code changes. This is similar in spirit to how our ETL will create site records for any new site we see rather than requiring the site to already exist. Thoughts?

I do think we'll have a conventional, mostly fixed set of scales for SFS and maybe another similar (but not the same) set for Nextstrain (when we eventually move Fauna → ID3C).

Reply:
Your comparison to the id3c etl process for creating new site makes sense. Perhaps we would want to restrict possible entries within the etl processes like we do for `etl clinical` ?

Reply:
The locations catalog is mostly manually managed by our code in the customizations repo. Most of the ETL processes don't create new locations, they only associate new encounters with existing locations. The exception is locations of scale `address`, which are created by the enrollments ETL. So we've effectively restricted scales that way. :-)